src/libs/ReportUtils.ts

```diff
@@ -621,17 +621,22 @@ function isDraftExpenseReport(report: OnyxEntry<Report> | EmptyObject): boolean
 }
 
 /**
- * Given a collection of reports returns them sorted by last read
+ * Checks if the supplied report has a common policy member with the array passed in params.
  */
-function sortReportsByLastRead(reports: OnyxCollection<Report>, reportMetadata: OnyxCollection<ReportMetadata>): Array<OnyxEntry<Report>> {
-    return Object.values(reports ?? {})
-        .filter((report) => !!report?.reportID && !!(reportMetadata?.[`${ONYXKEYS.COLLECTION.REPORT_METADATA}${report.reportID}`]?.lastVisitTime ?? report?.lastReadTime))
-        .sort((a, b) => {
-            const aTime = new Date(reportMetadata?.[`${ONYXKEYS.COLLECTION.REPORT_METADATA}${a?.reportID}`]?.lastVisitTime ?? a?.lastReadTime ?? '');
-            const bTime = new Date(reportMetadata?.[`${ONYXKEYS.COLLECTION.REPORT_METADATA}${b?.reportID}`]?.lastVisitTime ?? b?.lastReadTime ?? '');
+function hasReportCommonPolicyMemberWithArray(report: Report, policyMembersAccountIDs: string[]) {
```

Comment:
This might be cleaner name since the members of the report are called participants.
```suggestion
function hasParticipantInArray(report: Report, policyMembersAccountIDs: string[]) {
```